player/lua.c

```diff
@@ -1085,6 +1085,60 @@ static int script_readdir(lua_State *L)
     return 1;
 }
 
+static int script_stat(lua_State *L)
```

Comment:
Is this not something you can already do well enough via lua's standard library (I didn't check it out but it sounds like it should)? My opinion in general is that there's no point in adding mpv APIs for things which the language/setup supports natively.

Reply:
I don't see any mention of an `os.stat` for Lua. Maybe some extensions or LuaJIT provides them, but based on my knowledge there is no way to cross-platformly `stat` from Lua.

Reply:
From a quick search I couldn't find anything either.